### DISCUSSION TITLE
reset map

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#514] [Q&A] [answered]
Dear all,

is there any possibility we reset map display in leafmap, for example after some zoom in  we can able to use a tool to reset the display .
example in plotly :
![image](https://github.com/opengeos/leafmap/assets/141318843/632d8ca2-5570-4ab9-aa47-f32a68d8ff75)

thanks

**Comment by giswqs** [comment, marked as answer]
Use `fig.set_center()`. See https://leafmap.org/plotlymap/#leafmap.plotlymap.Map.set_center

**Reply by rh-afk** [reply]
thanks @giswqs , I try to use [https://leafmap.org/leafmap/#leafmap.leafmap.Map.set_center](url) but I got error 
![image](https://github.com/opengeos/leafmap/assets/141318843/fab7d113-99ed-4653-94bc-105fceaf7e80)
Is that something I need to specify and also is there any tool for saving the plot or map into file png etc like we have in plotly plot.

thanks

**Reply by giswqs** [reply]
Please read the API reference. `set_center(self, lon, lat, zoom=None)`
https://leafmap.org/leafmap/#leafmap.leafmap.Map.set_center

**Reply by rh-afk** [reply]
thanks @giswqs , unfortunately there is any toolbar for reset in the left of map and also strange location show up in the map
![image](https://github.com/opengeos/leafmap/assets/141318843/630614d0-860a-4ecb-b704-4fadedfece42)

**Reply by giswqs** [reply]
lon, lat
not lat, lon
